gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -176,14 +183,19 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin with Loggin
   }
 }
 
-private[glutenproject] class GlutenExecutorPlugin extends ExecutorPlugin {
+private[glutenproject] class GlutenExecutorPlugin extends ExecutorPlugin with Logging {
   private var executorEndpoint: GlutenExecutorEndpoint = _
   private val taskListeners: Seq[TaskListener] = Array(TaskResources)
 
   /** Initialize the executor plugin. */
   override def init(ctx: PluginContext, extraConf: util.Map[String, String]): Unit = {
     val conf = ctx.conf()
-
+    if (!conf.getBoolean(GLUTEN_ENABLE_KEY, defaultValue = true)) {
+      logWarning(
+        "Configured to not enabled Gluten, but gluten added in `spark.plugins`! " +
```

Comment:
ditto. but specifically add executor in log.

Reply:
Thanks for review, address those.